src/services/PlanManager.js

```diff
@@ -85,11 +85,17 @@ function getInstance(plans) {
         pricingBillingFrequency = BillingCycle.MONTHLY;
       }
 
+      const undiscountedPrice = (singleSitePrice * pricing.licenses);
```

Comment:
Shouldn't the `undiscountedPrice` be calculated based on `DiscountsModel`? Here we are calculating for `RELATIVE` only, but later on we are diving by `ABSOLUTE` if asked. Are you sure this is the behavior we want?